### PR TITLE
Grid placeholders

### PR DESCRIPTION
**Pull request by derpdead**
# Description

- Added placeholders to each Grid along with translations
- Added placeholders for tabs when Grid is covered by it
- Added placeholders to Widgets

For all of these places is used one component `Placeholder`

Issue #643 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [ ] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: modules/@ergonode/category-trees/src/locales/pl_PL.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    "advancedFilter": {
+      "label": "Drzewo kategorii"
+    },
+    "grid": {
+      "placeholderTitle": "Brak drzew kategorii",
+      "placeholderSubtitle": "W systemie nie ma drzew kategorii, stwórz pierwsze."
+    },
+    "messages": {
+        "deleteTitle": "",
```
Comment (bleto) [inline modules/@ergonode/category-trees/src/locales/pl_PL.json:21]: it should be an empty string?

File: modules/@ergonode/core/src/models/mappers/gridDataMapper.js [hunk 2]
```diff
@@ -156,23 +156,23 @@ export function getDefaultDataFromQueryParams($route) {
...
             page = DEFAULT_PAGE,
             itemsPerPage = DATA_LIMIT,
-            filter = '',
-            advancedFilter = '',
-            field = '',
-            order = '',
+            filter,
+            advancedFilter,
+            field,
+            order,
```
Comment (bleto) [inline modules/@ergonode/core/src/models/mappers/gridDataMapper.js:162]: Default values were a better solution. 
Some properties have default values here and the rest is conditioned in the function arguments. 
two different approaches, mixed up.

File: modules/@ergonode/media/src/components/Forms/ResourceRelationsForm.vue [hunk 3]
```diff
@@ -28,25 +28,17 @@
...
-        <ListPlaceholder
-            :orientation="horizontalOrientation"
-            title="Nothing to see here"
-            subtitle="Here you can see information about relations in products"
-            :bg-url="require('@UI/assets/images/placeholders/comments.svg')" />
-    </div>
+    <TabBarNoDataPlaceholer
+        v-else
+        title="Nothing to see here"
+        subtitle="Here you can see information about relations in products" />
```
Comment (bleto) [inline modules/@ergonode/media/src/components/Forms/ResourceRelationsForm.vue:34]: in this module, you already use `i18n` translations. This text should be replaced by the placeholder `$t()`.

File: modules/@ergonode/core/src/models/mappers/gridDataMapper.js (hunk 2 above)
Comment (derpdead) [reply modules/@ergonode/core/src/models/mappers/gridDataMapper.js:162]: It cannot lend in defaults and being exported as whole object. Query might have part of those fields. I would stick to conditions cuz defaults arguments are not set when any value is null and they might be null.

File: modules/@ergonode/category-trees/src/locales/pl_PL.json (hunk 1 above)
Comment (derpdead) [reply modules/@ergonode/category-trees/src/locales/pl_PL.json:21]: I was copying it, sorry for missing. I've added missing translations.

**Review by bleto — APPROVED** [review]

**Pull request merged**